.circleci/config.yml

```diff
@@ -1,93 +1,65 @@
-version: 2
+version: 2.1
+orbs:
+  docker: circleci/docker@0.5.19
+  django-tools: pennlabs/django-tools@0.0.1
+  helm-tools: pennlabs/helm-tools@0.1.4
+
+branch-filters: &branch-filters
+  filters:
+    branches:
+      only:
+        - master
 jobs:
-  build:
-    working_directory: ~/penncourses
-    docker:
-      - image: circleci/python:3.7-node
-        environment:
-          PIPENV_VENV_IN_PROJECT: true
-          DATABASE_URL: mysql://root:password@127.0.0.1:3306/circle_test
-      - image: circleci/mysql:latest
-        command: --default-authentication-plugin=mysql_native_password
-        environment:
-          MYSQL_ROOT_PASSWORD: password
-          MYSQL_DATABASE: circle_test
+  lint:
+    working_directory: ~/pennlabs
+      docker:
+          - image: circleci/node:10
```

Comment:
your yaml indentation is mucked up here. check template:
https://github.com/pennlabs/template-react-next/blob/master/.circleci/config.yml#L15